views/plugin/electron/customizesettingswindow.html

```diff
@@ -0,0 +1,28 @@
+<div class="flex-fill w-100 d-flex flex-column">
+  <div class="d-flex modal-body">
+    <customizesettings changed="ctrl.changed"></customizesettings>
```

Comment:
Can we combine the window with the "form" directive here rather than having two separate directives? I don't think we'll ever want to put `customizesettings` anywhere not in the window.

Reply:
I can, this is an artifact of initially having this UI in the Settings window as a settings plugin then moving it to a window later. I can combine them again, either way is fine by me.